src/locales/default.json

```diff
@@ -67,6 +67,10 @@
   "quorumReached": "quorum reached",
   "options": "Option(s)",
   "votingPower": "Your voting power",
+  "reason": {
+    "label": "Reason",
```

Comment:
`Reason` as label seems to limiting. Instead I would call it `Comment`

Reply:
Won't this confuse user with comment plugin? And the name `reason` are referenced in `snapshot.js vote()`(same as field in wallet signing UI)

Reply:
No I think it is clear the user is leaving the comment on their vote. I will discuss further with Fabien about this